app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,22 +1,18 @@
 import React, { useState } from "react";
 
 import { Alert } from "neetoui";
+import PropTypes from "prop-types";
 
 import notesApi from "apis/notes";
 
-const DeleteAlert = ({
-  refetch,
-  onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
-}) => {
+const DeleteAlert = ({ refetch, onClose, selectedNote, setSelectedNote }) => {
   const [deleting, setDeleting] = useState(false);
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      await notesApi.destroy({ ids: [selectedNote.id] });
```

Comment:
The state name is `selectedNote`. So it's just one note. So why do we need to pass in `ids` to `notesApi.destroy`?

I believe this is an issue with the naming of `selectedNote`. What was the issue with the previous naming?

Reply:
API takes array of ids to destroy, I changed the previous naming because we are only deleting one note at time. The original code was written for a multi select table, but in new UI we are using List which allows deletion of one note at a time. It helped in removing one state variable as both edit and delete can be facilitated in single variable.

Reply:
@mayanksharma4132 https://www.loom.com/share/d33d1230a4a84efbbf8ed81aba107e08

Reply:
@yedhink https://www.loom.com/share/7bd18440de284f598cbc756ffc148fdf

Reply:
@mayanksharma4132 

- You can add a destroy action in backend to delete a single note - you can rename the APIs too. But this is only for this particular case.
- You can make PR review changes in any of the upcoming PRs.